other_projects_that_use_sympy.tex

```diff
@@ -0,0 +1,69 @@
+There are several projects that use SymPy as a library for implementing
+a part of their project, or even as a part of back-end for their app as
+well.
+
```

Comment:
This empty line doesn't leaves a blank line, see [this](https://www.sharelatex.com/learn/Line_breaks_and_blank_spaces#Line_breaks).